app/views/prison/cancellations/new.erb

```diff
@@ -0,0 +1 @@
+<%= render 'prison/visits/processed' %>
```

Comment:
Is this used? Looks related to the rendering of cancellation errors, however there is no action or route to `cancellations#new`

Reply:
that is indeed used, it is what is rendered when validation errors occurs, pretty standard rails pattern:
if `#create` actions has validation errors render the `#new` with validation errors displayed.